src/js/controller/controller.js

```diff
@@ -289,7 +289,7 @@ define([
                 if (_model.get('playOnViewable')) {
                     if (viewable) {
                         _autoStart();
-                    } else if (utils.isMobile()) {
+                    } else if (utils.isMobile() && model.get('mute')) {
```

Comment:
@robwalch I'm still a bit confused. So it's not enough to check the volume? We would want to set playOnViewable to false if volume is no longer muted ?